src/api/forms/components/FormProperties.vue

```diff
@@ -132,6 +140,11 @@ export default {
             }
 
             return 'Cancel';
+        },
+        shouldHideCancelButton() {
+            return this.model.buttons
+                && this.model.buttons.cancel
+                && this.model.buttons.cancel.hide === true;
```

Comment:
Might be good to use optional chaining here? `this.model?.buttons?.cancel?.hide`

Reply:
👍